package.json

```diff
@@ -7,7 +7,7 @@
     "url": "https://github.com/ministryofjustice/help-with-prison-visits-database.git"
   },
   "engines": {
-    "node": "16.x.x",
+    "node": "18",
     "npm": "8"
```

Comment:
Wondering if we should switch to NPM version 9 🤔? Looks to be the default installed with `nvm`:
```
$ nvm use 18
Now using node v18.14.0 (npm v9.3.1)
```

Could also add an `.nvmrc` with `18`? I've been experimenting with `ohmyzsh` plugin for `nvm` and if you add [this config](https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/nvm#nvmrc-autoload) to `.zshrc` it auto switches version for you

Reply:
Done!